src/util/currency.spec.js

```diff
@@ -273,6 +273,19 @@ describe('currency', () => {
         testCurrency(inputs, ccy, 'de-DE', outputs);
       });
     });
+
+    it('should place minus sign in front', () => {
```

Comment:
Maybe we can make this test case more intuitive. A suggestion would be adding a `describe` statement as a parent of it like so:
```javascript
describe('handling negative amounts', () => {
  it('should place a minus sign in front of the currency', () => {
    { ... }
  });
});
```
WDYT?  🙂

Reply:
Much better =)